package.json

```diff
@@ -79,7 +84,10 @@
     "usePathForSuiteName": "true"
   },
   "dependencies": {
+    "asn1.js-rfc2560": "^5.0.0",
+    "asn1.js-rfc5280": "^3.0.0",
```

Comment:
you need both rfcs?

Reply:
They're both used by the OCSP code I'm importing.